ZotBinsAPI.py

```diff
@@ -282,7 +289,94 @@ def get_obervation_stats():
         print(e)
         return str(e)
 
+@app.route('/mocked/observation/get', methods=['GET'])
+def mocked_get_observation():
+    """ 
+    An API endpoint that mocks observation GET requests.
+
+    Used for web and mobile applications to simulate
+    an identical GET request for real data.
+
+    Must pass all required params.
+
+    Possible sensor types = ["B", "D", "W", "DI"]
+        "B"  - Breakbeam
+        "D"  - Distance (ultrasonic sensor)
+        "W"  - Weight (load cell)
+        "Di" - Diversion Rate
+
+    @param   sensor_id, format = "ZBin" + [SENSOR_ID] + [SENSOR_TYPE]
+    @param   start_timestamp, format = "YYYY-MM-DD HH:MM:SS"
+    @param   end_timestamp, format example = "2020-02-01 9:30:01"
+    @return  JSON-ified array of mocked datapoints
+    """
+    try:
+        con = pymysql.connect(config.host, config.user, config.pw, config.db, cursorclass=pymysql.cursors.DictCursor)
+        ret = []
+        if request.method == 'GET':
+            sensor_id = request.args.get("sensor_id")
+            start_timestamp = request.args.get("start_timestamp")
+            end_timestamp = request.args.get("end_timestamp")
+
+            """
+            Perform a quick null check and return
+            useful info to the caller so that they
+            know what fields they should be adding.
+            
+            TODO: Probably should have this null check 
```

Comment:
isn't below code block addressing this TODO?

Reply:
This TODO is for ALL methods. Glancing over all the other API's, the null checking that executes is `if sensor_id is not None or start_timestamp is not None or end_timestamp is not None:`

However, I think it's preferable to do it line-by-line like here so that we could return relevant error messages to the API caller. For example, "need to specify a sensor_id", "can't leave start_timestamp null", etc.